pkg/controller/config/config_controller.go

```diff
@@ -116,6 +117,12 @@ func (r *ReconcileConfig) Reconcile(request reconcile.Request) (reconcile.Result
 		log.Error(err, "failed to get Cluster Network CR")
 		return reconcile.Result{Requeue: true}, err
 	}
+	if request.Name == clusterConfig.Name && r.appliedClusterConfig != nil {
```

Comment:
How about make this change in a different patch

Reply:
Sure. Will create a new PR.